deploy/test-environments/fleet_api/src/install_cnvm_integration.py

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env python
+"""
+This script installs CNVM AWS integration
+
+The following steps are performed:
+1. Create an agent policy.
+2. Create a CNVM AWS integration.
+3. Create a CNVM bash script to be deployed on a host.
```

Comment:
This script generates the config.json file, which is utilized in the deployment of the stack (`deploy/cloudformation`)